bom/pom.xml

```diff
@@ -470,6 +472,18 @@
             <artifactId>c3p0</artifactId>
             <version>${version.c3p0}</version>
          </dependency>
+
+         <dependency>
+            <groupId>com.zaxxer</groupId>
+            <artifactId>HikariCP</artifactId>
+            <version>${version.hikaricp}</version>
+         </dependency>
+
+         <dependency>
+            <groupId>org.jboss.slf4j</groupId>
+            <artifactId>slf4j-jboss-logging</artifactId>
```

Comment:
@slaskawi I was following the lead of the C3P0 dependencies here and placed the dependencies in both bom and parent.